ui/src/app/applications/components/application-node-info/application-node-info.tsx

```diff
@@ -57,6 +57,13 @@ export const ApplicationNodeInfo = (props: {
                 hostNames = (status.loadBalancer.ingress || []).map((item: any) => item.hostname || item.ip).join(', ');
             }
             attributes.push({title: 'HOSTNAMES', value: hostNames});
+        } else if (props.node.kind === 'AnalysisRun') {
```

Comment:
This would be the first case of special handling of `AnalysisRun` in the codebase. This is a great convenience feature, but I think we should avoid introducing special handling and instead rely on extensions for functionality like this. Even though this is a relatively inconsequential change, it sets a precedent.

@alexmt WDYT?

Reply:
I agree with you, although to my understanding, the extension feature controls only the "More" tab. So it's a click away from the summary.

I can rewrite this in a more generic way, as the message I'm displaying here is also wrapped in the `HealthStatus.message` field. So it could be displayed for all resources providing a health state (some already do), not only AnalysisRun.

Reply:
had to cast from ResourceNode to ResourceTreeNode (but I saw we're already doing that in resource-details.tsx), but now the health message is displayed to all non controlled resources that have it.

Reply:
ha, only now I realised you tagged @alexmt not myself 😂 in the comment above

too many Alexes